File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// Magic8Ball SDK exists test

require_once __DIR__ . '/../magic8ball_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = Magic8BallSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# Magic8Ball SDK exists test

require "minitest/autorun"
require_relative "../Magic8Ball_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = Magic8BallSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
